content/en/docs/examples/mesh-expansion/_index.md

```diff
@@ -1,9 +0,0 @@
----
```

Comment:
If this file is deleted, are any links pointing to it (off of istio.io) now broken? Will there be a redirect?

Reply:
alias directive are added in the new destination files which inject redirects. The tool to move pages around the site takes care of injecting those aliases.

Reply:
And I used the wonderful `mvpage` tool to move the pages to ensure that happened. Thanks again @geeknoid